exercises/pig-latin/tests/pig-latin.rs

```diff
@@ -113,6 +113,12 @@ fn test_word_beginning_with_xr() {
     assert_eq!(pl::translate("xray"), "xrayay");
 }
 
+#[test]
+#[ignore]
+fn test_word_word_beginning_with_y_and_a_preceding_consonant() {
```

Comment:
A small nitpick: perhaps you meant to name the test function `test_word_beginning_with_y_and_a_preceding_consonant`, without an extra `_word_`?

Also I believe the test naming policy on this track is that you take a description from the `canonical-data.json` and replace all spaces ` ` with the `_`, so the name would be `test_y_is_treated_like_a_vowel_at_the_end_of_a_ consonant_cluster` (Note that it is just my comment and you do not have to rename your function unless the maintainer says so)

Reply:
@ZapAnton both good points. I'll rename it when I'm next at my computer.